pegasus/sites.v3/code.org/public/css/promote.css

```diff
@@ -0,0 +1,87 @@
+#promote-page h1 {
+  margin-bottom: 15px;
+}
+
+#resources-section {
+  margin-top: -50px;
+}
+
+#youtube-section h2 {
+  margin-top:  0px;
+}
+
+.col-18-promote-tools {
+  height: 260px;
+  margin: 1%;
+}
+
+.left {
+  float: left
+}
+
+a.whitefooterlink {
+  text-decoration: none !important;
```

Comment:
I try to avoid using `!important` as much as possible. Where else is it getting text-decoration from and can that class be more specific to not affect the footer links? It may not be trivial or possible, but just a thought.